src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -207,7 +207,9 @@ class ArrayField extends React.Component {
       <div className={itemsNeeded ? 'schemaform-review-array-warning' : null}>
         {title && (
           <div className="form-review-panel-page-header-row">
-            <h5 className="form-review-panel-page-header">{title}</h5>
+            {title ? (
+              <h5 className="form-review-panel-page-header">{title}</h5>
+            ) : null}
```

Comment:
This could just be:

```
{title && <h5 ...>{title}</h5>}
```

Reply:
Removed since checking the `title` value is repetitious (checked in the outer wrapper).